common/src/primitives/id.rs

```diff
@@ -42,9 +42,16 @@ impl From<Uint256> for H256 {
     }
 }
 
-#[derive(Debug, Clone, PartialEq, Eq, Encode, Decode)]
+impl serde::Serialize for H256 {
```

Comment:
We probably also need deserialize here + tests for both

Reply:
Sure will add